### PR TITLE
Updated all Spanish translations

### PR DESCRIPTION
**Pull request by rmcpantoja**
Description:
Fixed some messages, added the new descriptions and messages for accessibility and more.

### CODE REVIEW

**Comment by JeffersonRussell** [comment]
Some changes are not yet in the 0000.lang file.
"Buscar actualizaciones" is too long and there is not enough space.
The descriptions have errors and lack of context.

Let me update the descriptions and if you want add the messages for accessibility when they are in the 0000.lang file

**Comment by rmcpantoja** [comment]
about "buscar actualizaciones", I feel that it is the correct term to refer to (check for updates) since "actualizar" can generate some confusion, and can be interpreted in cases, as a order to the user.

**Comment by JeffersonRussell** [comment]
> about "buscar actualizaciones", I feel that it is the correct term to refer to (check for updates) since "actualizar" can generate some confusion, and can be interpreted in cases, as a order to the user.

Yes, but there is not enough space. Another possible option could be "Actualizaciones" but I think it is less accurate than "Actualizar"

File: Langs/080A.lang [hunk 1]
```diff
@@ -8,9 +8,12 @@ Large=18
 Medium=10
 Small=8.5
 [Strings]
+Enable enanced accessibility?=¿Activar accesibilidad mejorada?
+This new Enhanced Accessibility functionality is designed for the visually impaired, in which most of the program interface can be used by voice and keyboard shortcuts. Activate?=Esta nueva funciòn de la accesibilidad mejorada está diseñada para las personas con discapacidad visual, en la que la mayor parte de la interfaz del programa se puede utilizar mediante voz y atajos de teclado. ¿Activar?
+The compatibility results were copied to the clipboard.Resultados de compatibilidad del sistema copiados al portapapeles.
```
Comment (micwoj92) [inline Langs/080A.lang:13]: Misses = sign here, and in other files too.

**Comment by micwoj92** [comment]
@rmcpantoja this pr has now many conflicts because of changes from @JeffersonRussell that were accepted. Do you think there is still something to add or would be it better to open new pull request?

**Pull request closed without merging**